### ISSUE TITLE
unknown message

### ISSUE THREAD

**Issue by javipalanca** [DinoThor/PyJabber#6]
There seems to be an error with the ping iq messages.

```
2024-05-09 18:08:55.395 | DEBUG    | pyjabber.network.XMLProtocol:data_received:97 - Data received: <iq id=":hyBXT6OCXTkjcG8vWi6s" type="get"><ping xmlns="urn:xmpp:ping"/></iq>
2024-05-09 18:08:55.396 | ERROR    | pyjabber.network.XMLProtocol:data_received:117 - SAXParser Error parsing XML: <unknown>:1:1077: parsing finished
```

**Comment by DinoThor** [comment]
https://xmpp.org/extensions/xep-0199.html

**Comment by DinoThor** [comment]
Implemented ping xep in f43e7f3885fbf20540d890e9082032bb8a2df151
